lib/vuln/php_easter_egg/engine.py

```diff
@@ -0,0 +1,201 @@
+    except Exception as e:
+        print(e)
+        return False
+
+
+def __php_easter_eggs_vuln(target, port, timeout_sec, log_in_file, language, time_sleep,
+                     thread_tmp_filename, extra_requirements, socks_proxy, scan_id, scan_cmd):
+    if php_easter_eggs_vuln(target, port, timeout_sec, log_in_file, language, time_sleep,
+                      thread_tmp_filename, extra_requirements, socks_proxy, scan_id, scan_cmd):
+        info(messages(language, "found").format(target, "PHP easter Egg!", php_easter_egg))
+        __log_into_file(thread_tmp_filename, 'w', '0', language)
+        data = json.dumps({'HOST': target, 'USERNAME': '', 'PASSWORD': '', 'PORT': port, 'TYPE': 'php_easter_eggs_vuln_scan',
+                           'DESCRIPTION': messages(language, "found").format(target, "PHP easter Egg!", php_easter_egg), 'TIME': now(),
+                           'CATEGORY': "scan",
+                           'SCAN_ID': scan_id, 'SCAN_CMD': scan_cmd})
+        __log_into_file(log_in_file, 'a', data, language)
+        return True
+    else:
+        return False
+
+
+def start(target, users, passwds, ports, timeout_sec, thread_number, num, total, log_in_file, time_sleep, language,
+          verbose_level, socks_proxy, retries, methods_args, scan_id, scan_cmd):  # Main function
+    if target_type(target) != 'SINGLE_IPv4' or target_type(target) != 'DOMAIN' or target_type(target) != 'HTTP':
+        # requirements check
+        new_extra_requirements = extra_requirements_dict()
+        if methods_args is not None:
+            for extra_requirement in extra_requirements_dict():
+                if extra_requirement in methods_args:
+                    new_extra_requirements[extra_requirement] = methods_args[extra_requirement]
+        extra_requirements = new_extra_requirements
+        if ports is None:
+            ports = extra_requirements["php_easter_eggs_vuln_ports"]
+        if target_type(target) == 'HTTP':
+            target = target_to_host(target)
+        threads = []
+        total_req = len(ports)
+        thread_tmp_filename = '{}/tmp/thread_tmp_'.format(load_file_path()) + ''.join(
+            random.choice(string.ascii_letters + string.digits) for _ in range(20))
+        __log_into_file(thread_tmp_filename, 'w', '1', language)
+        trying = 0
+        keyboard_interrupt_flag = False
+        for port in ports:
+            port = int(port)
+            t = threading.Thread(target=__php_easter_eggs_vuln,
+                                 args=(target, int(port), timeout_sec, log_in_file, language, time_sleep,
+                                       thread_tmp_filename, extra_requirements, socks_proxy, scan_id, scan_cmd))
+            threads.append(t)
+            t.start()
+            trying += 1
+            if verbose_level > 3:
+                info(
+                    messages(language, "trying_message").format(trying, total_req, num, total, target, port, 'php_easter_eggs_vuln_scan'))
```

Comment:
@aman566 please use the same name you started with 'php_easter_egg_vuln_scan' (not 'eggs')